android/src/main/java/ly/count/android/sdk/react/CountlyReactNative.java

```diff
@@ -793,6 +793,22 @@ public void setUserData(ReadableArray args, Promise promise) {
         promise.resolve("Success");
     }
 
+    @ReactMethod
+    public void setProperties(ReadableArray args, Promise promise) {
+        Countly.sharedInstance();
+        ReadableMap userData = args.getMap(0);
+        Map<String, Object> userDataObjectMap = userData.toHashMap();
+
+        for (Map.Entry<String, Object> entry : userDataObjectMap.entrySet()) {
+            String key = entry.getKey();
+            Object value = entry.getValue();
+            userDataObjectMap.put(key, value);
+        }
+        Countly.sharedInstance().userProfile().setProperties(userDataObjectMap);
+        Countly.sharedInstance().userProfile().save();
```

Comment:
mb saving is not needed. Because we already do internal checks to triggering savings.